### ISSUE TITLE
[IMP] ungrouped-isinstances: Use isinstance(value, (class1, class2)) instead of isinstance(value, class1) or isinstance(value, class2)

### ISSUE THREAD

**Issue by moylop260** [PyCQA/pylint#968]
I wish create a new checker to avoid a ungrouped isinstance.

E.g.
```python
# bad
if isinstance(value, class1) or isinstance(value, class2):
   pass

# good
if isinstance(value, (class1, class2)):
    pass
```

The question is: Is a good approach to pylint project or I should create it from my own project extensions?

**Comment by PCManticore** [comment]
I don't think it's that worth it for now. Is this pattern of checking with more than two classes that common?

**Comment by moylop260** [comment]
> Is this pattern of checking with more than two classes that common?

We could take pylint examples like as a sample:

Grouped cases:
[pylint/checkers/base.py#L102-L103](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/base.py#L102-L103)
[pylint/checkers/base.py#L113](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/base.py#L113)
[pylint/checkers/base.py#L398](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/base.py#L398)
[pylint/checkers/base.py#L566](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/base.py#L566)
[pylint/checkers/base.py#L582](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/base.py#L582)
[pylint/checkers/base.py#L586](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/base.py#L586)
[pylint/checkers/base.py#L777](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/base.py#L777)
[pylint/checkers/base.py#L793](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/base.py#L793)
[pylint/checkers/base.py#L891](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/base.py#L891)


Not grouped cases:
[pylint/checkers/base.py#L186-L187](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/base.py#L186-L187)
[pylint/checkers/utils.py#L247-L249](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/utils.py#L247-L249)
[pylint/checkers/variables.py#L464-L465](https://github.com/PyCQA/pylint/blob/5d40a07684be7ed6dbaa54e49c3d436005a533bf/pylint/checkers/variables.py#L464-L465)

Do you agree if I create the PR like as `extensions`?

**Comment by The-Compiler** [comment]
FWIW this is a little style issue I'd like to get consistent, but always forget about it - so I'd personally like to have a checker for this... just my $0.02 :wink:

**Comment by PCManticore** [comment]
If you find this useful, then I am okay having it as an extension.
